src/videotilecontroller/DefaultVideoTileController.ts

```diff
@@ -56,8 +57,21 @@ export default class DefaultVideoTileController implements VideoTileController {
     tile.bindVideoElement(videoElement);
   }
 
-  unbindVideoElement(tileId: number): void {
-    this.bindVideoElement(tileId, null);
+  unbindVideoElement(tileId: number, cleanUpVideoElement: boolean = true): void {
+    const tile = this.getVideoTile(tileId);
+    if (tile === null) {
+      this.logger.warn(`Ignoring video element unbinding for unknown tile id ${tileId}`);
+      return;
+    }
+    if (cleanUpVideoElement) {
```

Comment:
nit: Can we remove the else layer?
```
this.logger.info(`Unbinding the video element`);
tile.bindVideoElement(null);
if (cleanUpVideoElement) {
 this.logger.info(`Unbinding and cleaning up the video element`);
 DefaultVideoTile.disconnectVideoStreamFromVideoElement(tile.stateRef().boundVideoElement, false);
}
```

Reply:
`tile.stateRef().boundVideoElement` will return a wrong object after `tile.bindVideoElement(null)`?

Reply:
```js
this.logger.info(`Unbinding the video element`);

const videoElement = tile.stateRef().boundVideoElement;
tile.bindVideoElement(null);
if (cleanUpVideoElement) {
  this.logger.info(`Cleaning up the video element`);
  DefaultVideoTile.disconnectVideoStreamFromVideoElement(videoElement, false);
}
```

Reply:
Yes, that is correct Kyu. After `tile.bindVideoElement(null)`, the `boundVideoElement` is `null` hence, have to maintain the ref before `bindVideoElement(null)` call.

Reply:
Also, I just wanted to avoid stale `videoElement` reference cause, if `cleanupVideoElement` is `false` then no use for the maintained ref. Since, that is not the default behavior is it fine to do that way then?

```
const videoElement = tile.stateRef().boundVideoElement;
tile.bindVideoElement(null);
if (cleanUpVideoElement) {
  DefaultVideoTile.disconnectVideoStreamFromVideoElement(videoElement, false);
}
```

Reply:
Looks like @simmkyu provided the code snippet already, so `videoElement` reference wont be used if `videoElement` is `false`, is that fine 😅 ?

Reply:
Not using `videoElement` in the `else` case should be fine. I don't have a strong preference between `else` vs without `else`.

Reply:
I will update as suggested I also do not have a preference, but the suggested looks more cleaner. Thanks!

Reply:
Updated the code, thanks!